src/alpaka/AlpakaDataFormats/BeamSpotAlpaka.h

```diff
@@ -0,0 +1,29 @@
+#ifndef AlpakaDataFormats_BeamSpot_interface_BeamSpotAlpaka_h
+#define AlpakaDataFormats_BeamSpot_interface_BeamSpotAlpaka_h
+
+#include "AlpakaCore/alpakaCommon.h"
+#include "DataFormats/BeamSpotPOD.h"
+
+#include <cstring>
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class BeamSpotAlpaka {
+  public:
+    BeamSpotAlpaka() = default;
+
+    BeamSpotAlpaka(BeamSpotPOD const* data, Queue& queue) : data_d{cms::alpakatools::allocDeviceBuf<BeamSpotPOD>(1u)} {
+      auto data_h{cms::alpakatools::createHostView<const BeamSpotPOD>(data, 1u)};
```

Comment:
Does `createHostView()` create a copy, or only a non-owning "View" that can be passed to `alpaka::memcpy()` in lieu of the `data` pointer?

Reply:
`createHostView()` creates a non-owning view.
One cannot passes the raw data pointer to `alpaka::memcpy()`.
Basically:
- the `alpakaHost/DeviceBuf `are owning (shared) pointers.
- with `alpaka::getPtrNative(myDeviceBuf)`, one gets a non-owning (raw) pointer.
- `alpakaHost/DeviceViews` are just wraps around raw pointers, allowing operations like `alpaka::memcpy()`, or also allowing to view a fraction of the owned data with a `alpakaSubview`.